src/core/manager.c

```diff
@@ -3568,14 +3540,6 @@ int manager_reload(Manager *m) {
 
         m->send_reloading_done = true;
         return 0;
```

Comment:
Missing `reloading = NULL;`. This function decreases `m->n_reloading` only when the above operation fails.